kubernetes-kit-starter/src/main/java/com/vaadin/kubernetes/starter/sessiontracker/SessionTrackerCookie.java

```diff
@@ -41,15 +41,18 @@ private SessionTrackerCookie() {
     public static void setIfNeeded(HttpSession session,
             HttpServletRequest request, HttpServletResponse response) {
         Optional<Cookie> clusterKeyCookie = getCookie(request);
-        if (!clusterKeyCookie.isPresent()) {
+        if (clusterKeyCookie.isEmpty()) {
             String clusterKey = UUID.randomUUID().toString();
             session.setAttribute(CurrentKey.COOKIE_NAME, clusterKey);
-            response.addCookie(new Cookie(CurrentKey.COOKIE_NAME, clusterKey));
+            Cookie cookie = new Cookie(CurrentKey.COOKIE_NAME, clusterKey);
+            cookie.setHttpOnly(true);
+            cookie.setPath("/");
+            cookie.setAttribute("SameSite", "Strict");
```

Comment:
Setting the SameSite  attribute seesm a good idea, but I wonder if this can break the application in some case (e.g. old browsers).
It would be nice if the value could be configurable, like Spring does with `server.servlet.session.cookie.same-site` property. But this change may require additional effort.

Reply:
Made the SameSite attribute configurable by property. Example:
```
vaadin:
  kubernetes:
    cluster-key-cookie-same-site: strict
```